azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/models/ChatCompositeEventCode.kt

```diff
@@ -6,6 +6,9 @@ package com.azure.android.communication.ui.chat.models
 import com.azure.android.core.util.ExpandableStringEnum
 
 internal class ChatCompositeEventCode : ExpandableStringEnum<ChatCompositeEventCode?>() {
+    companion object {
+        val CHAT_EVICTED = fromString("chatEvicted", ChatCompositeEventCode::class.java)
```

Comment:
renamed to `CHAT_REMOVED`

Reply:
hey I don't think we should use this since people might get confused with chat thread deleted. I don't think we should have event code for this, shouldn't we be using event handler instead?